src/Console/WebsocketRunnerCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Concrete5GraphqlWebsocket\Console;
+
+use Concrete\Core\Config\Repository\Repository;
+use Concrete\Core\Console\Command;
+use Concrete5GraphqlWebsocket\SchemaBuilder;
+use Siler\GraphQL as SilerGraphQL;
+
+class WebsocketRunnerCommand extends Command
+{
+    protected $description = 'Start a websocket server';
+
+    protected $signature = <<<'EOT'
+gws:start
+    {port : the websocket port number}
+EOT
+    ;
+
+    public function handle(Repository $config)
+    {
+        $port = (int) $this->input->getArgument('port');
+        if ($port < 1 || $port > 65535) {
+            $this->output->error('The port number must be an integer between 1 and 65535');
+
+            return 1;
+        }
+        $serverIP = $config->get('concrete5_graphql_websocket::websocket.server_ip');
+        $this->output->write("Checking availability of port {$port} on {$serverIP}... ");
+        $connection = @fsockopen($serverIP, $port);
+        if (is_resource($connection)) {
+            fclose($connection);
+            $this->output->error("no server found!\n");

```

Comment:
$this->output->error("server already exists on this port!\n");

Reply:
Whoops, you are right of course! Done.